datafusion/src/physical_plan/planner.rs

```diff
@@ -283,7 +322,7 @@ impl DefaultPhysicalPlanner {
                 Ok(Arc::new(WindowAggExec::try_new(
                     window_expr,
                     input_exec.clone(),
-                    physical_input_schema,
+                    input_exec.schema(),
```

Comment:
we should be able to take ownership of `physical_input_schema` here instead of calling into `schema()` right?